test/integration/targets/zypper_repository/tasks/zypper_repository.yml

```diff
@@ -125,3 +125,18 @@
     priority: 100
     auto_import_keys: true
     state: "present"
+
+- name: add docker repo with params
```

Comment:
Could you please add the repo again and `assert` no change has occured?

Reply:
When you said "add the repo again" are you referring to re run again by amending this patch?

Reply:
@gustavmb apologize I should have been clearer

Currently you have
1) name: add docker repo with params
2) name: check listrepo from custom root
3) assert

If you add
4) name: add docker repo with params (again, just duplicate the tets) 
5) assert no change


Then we ensure that the module is idempotent, ie running it again results in no change

Reply:
Done. I added a second  "add docker repo with params" for checking the idempotence :)